tt_metal/hw/ckernels/grayskull/common/inc/ckernel_globals.h

```diff
@@ -12,7 +12,11 @@ extern uint32_t cfg_state_id;
 extern uint32_t unp_cfg_context;
 extern uint32_t gl_alu_format_spec_reg;
 
-extern volatile uint32_t l1_buffer[16];
+extern volatile uint32_t tt_l1_ptr l1_buffer[16];
```

Comment:
should this be a pointer?

Reply:
This l1 buffer is used by the reduce op and I believe it meant to be static l1 memory, but Ill add @acejkov to confirm.

Reply:
yes, we need it for buda